content/docs/user-guide/deploying/docker.md

```diff
@@ -10,14 +14,50 @@ pip install mlem[docker]
 pip install docker
 ```
 
-## TODO
+## Description
+
+Deploying to a docker container involves 2 main steps:
+
+1. Build the docker image.
```

Comment:
Link to user-guide/building/docker

Reply:
+ maybe "(by running `docker build` under the hood)"

Reply:
done